qemu/qemu.go

```diff
@@ -2280,6 +2313,10 @@ type Knobs struct {
 
 	// IOMMUPlatform will enable IOMMU for supported devices
 	IOMMUPlatform bool
+
+	// UEFISupport will enable the use of UEFI/OVMF instead of Bios
+	// Assumes config.Bios points to an OVMF image
+	UEFISupport bool
```

Comment:
Ideally the UEFI support stuff would be a separate commit, but it's not a show stopper.

Reply:
Agreed. Done